packages/mikado_reborn/src/components/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../assets/styles/styles.scss';
+
+.mkr__checkbox {
+
+  background-color: map-get($colors, 'white');
+  border: 1px solid map-get($colors, 'neutral-20');
+  border-radius: 1rem;
+
+  &.medium {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 6px;
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: 0 0 0 0.5rem transparentize(map-get($colors, 'secondary'), 0.8);
```

Comment:
Can you use `$primary-secondary-20` in the sass tokens and make a `secondary-20` variable in color files please ? This one is missing but should be used here.